tests/test_numbers.py

```diff
@@ -252,6 +253,13 @@ def test_format_currency():
     assert (numbers.format_currency(1099.98, 'EUR', locale='nl_NL')
             != numbers.format_currency(-1099.98, 'EUR', locale='nl_NL'))
 
+    assert (numbers.format_currency(1099.98, 'JPY', locale='en_US',
+                                    currency_digits=True)
+            == u'\xa51,100')
+    assert (numbers.format_currency(1099.98, 'COP', u'#,##0.00', locale='es_ES',
```

Comment:
`currency_digits=False` is the default (you can see that in the diff).  The idea is to maintain syntactic and semantic backwards compatibility with older user code.